### DISCUSSION TITLE
Double dot .. notation

### DISCUSSION THREAD

**Discussion by Damecek** [jongpie/NebulaLogger#653] [General]
Hi, i've stumbled upon a weird notation in this code base, LogEntryEventBuilder.cls, uses double dot `..` which is not common. Frankly speaking I think it is a typo, but maybe it is correct. Just thought to leave it here if it is in the and a bug. Weirdly such code deploys ok.

E.g.:
```apex
Schema.LogEntryEvent__e..HttpRequestBody__c
Schema.LogEntryEvent__e..RestRequestBody__c
```

**Comment by jongpie** [comment]
@Damecek thanks for pointing this out! This is definitely a typo, and I'm fascinated that it's deployable. But it's unintentional, it should only have a single dot `.` - I'll fix this in the next release.
